TSOClient/tso.common/rendering/framework/io/InputManager.cs

```diff
@@ -54,6 +54,8 @@ public IFocusableUI GetFocus()
         /// <param name="keys"></param>
         public KeyboardInputResult ApplyKeyboardInput(StringBuilder m_SBuilder, UpdateState state, int cursorIndex, int cursorEndIndex, bool allowInput)
         {
+            if (!state.WindowFocused) { return null; }
```

Comment:
I might need to change this for volcanic 2.0 (which adds keyboard states for winforms game views)